scripts/scil_filter_streamlines_by_length.py

```diff
@@ -1,29 +1,39 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 import argparse
+import logging
 
-from nibabel.streamlines import load, save, Tractogram
+from dipy.io.stateful_tractogram import Space, StatefulTractogram
+from dipy.io.streamline import save_tractogram
 import numpy as np
 
 from scilpy.tracking.tools import filter_streamlines_by_length
-from scilpy.io.utils import (assert_inputs_exist, assert_outputs_exist,
-                             add_overwrite_arg)
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.utils import (assert_inputs_exist,
+                             assert_outputs_exist,
+                             add_overwrite_arg,
+                             add_reference,
+                             add_verbose_arg)
 
 
 def _build_args_parser():
     p = argparse.ArgumentParser(
         formatter_class=argparse.RawTextHelpFormatter,
         description='Filter streamlines by length.')
-    p.add_argument('in_tractogram', type=str,
+    p.add_argument('in_tractogram',
                    help='Streamlines input file name.')
-    p.add_argument('out_tractogram', type=str,
+    p.add_argument('out_tractogram',
                    help='Streamlines output file name.')
     p.add_argument('--minL', default=0., type=float,
                    help='Minimum length of streamlines. [%(default)s]')
     p.add_argument('--maxL', default=np.inf, type=float,
                    help='Maximum length of streamlines. [%(default)s]')
+    p.add_argument('--no_empty', action='store_true',
```

Comment:
Is there a use case where we would allow an empty output file ? If not, I would remove the argument and just return the debug message (or an exception)

Reply:
Some of our pipeline or collaborators still need this option.